did/didurl.go

```diff
@@ -0,0 +1,163 @@
+package did
```

Comment:
copyright

Reply:
None of the files in this lib have a copyright notice. We should add it in a separate PR.